gatsby-node.js

```diff
@@ -5,9 +5,10 @@ module.exports.createPages = async ({ graphql, actions }) => {
     const directorPageTemplate = path.resolve('./src/templates/directorPage.js')
     const res = await graphql(`
     query {
-        allContentfulPerson  {
+        allContentfulPerson(filter:{node_locale: { eq: "en-US"}}) {
```

Comment:
In the future can add a variable instead of a hardcode. The variable will be bound to the language switch. Sth like this: 
`allContentfulPerson(filter:{node_locale: { eq: "{locale}"}}) `